src/Runner.Worker/Container/ContainerHooks/HookResponse.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public class HookResponse
+    {
+        public ResponseContext Context { get; set; }
```

Comment:
Something to consider, we should think about parsing each hook response as a particular type, like PrepareJobResponse, and throwing if fields that are required are missing. May be easier to build on in the future. We do a lot of assuming things are stuff like strings currently, and i'm curious if that is going to cause us issues.

Reply:
I've got the same feeling, I'll look into splitting responses between hooks